docs/pages/upgrading/agent-managed-updates/bootstrap-ec2-agent.mdx

```diff
@@ -0,0 +1,249 @@
+ - Persists most flags you pass (e.g., -g, -p, -b) so re-running enable updates the saved settings.
+
+For full Managed Updates v2 instructions, see [Managed Updates forAgents (v2)](./agent-managed-updates.mdx).
+
+Groups & schedules: Every agent belongs to an update group (e.g., development, staging, prod).
+The cluster-side `autoupdate_config` resource defines when each group may update.
+Cloud or self-hosted clusters use the same schedule model, self-hosted clusters also set the desired
+version via `autoupdate_version`.
+
+## Define groups and schedules
+
+Pick simple, meaningful groups like `development`, `staging`, `production`. Then model your rollout windows
+and sequencing in `autoupdate_config`.
+
+```yaml
+# autoupdate_config.yaml
+kind: autoupdate_config
+metadata:
+  name: autoupdate-config
+spec:
+  agents:
+    mode: enabled
+    strategy: halt-on-error
+    schedules:
+      regular:
+        - name: development
+          days: ["Mon","Tue","Wed","Thu"]
+          start_hour: 4   # UTC
+        - name: staging
+          days: ["Mon","Tue","Wed","Thu"]
+          start_hour: 5
+          wait_hours: 24  # run a day later than development
+```
+
+This resource is used to controls the update schedule by defining different groups, sequence and upgrade window.
+
+Run the following command to create or update the resource:
+
+```code
+$ tctl create -f autoupdate_config.yaml
+```
+
+## Join Methods and Tokens
+
+Joining a Teleport cluster is the act of establishing trust between a new Teleport instance and all
+the existing instances already part of the Teleport cluster. At the end of the joining process,
+the Auth Service signs certificates for the joining instance. Those certificates represent the
+trust that was established. With them, the newly-joined instance can interact with the other Teleport instances.
+
+For bootstrapping EC2, the most appropriate methods are creating short-lived or long-lived tokens,
+or using delegated join with the IAM join method.
+
+### Creating join tokens
+
+Create a join token with an explicit TTL and the right service type.
+By default, `tctl tokens add` issues short-lived tokens (expiry is 30 minutes if you omit --ttl).
+
+```code
+$ tctl tokens add --type=node --ttl=1h --format=text
+```
+
+Long-lived option: a token resource with an explicit expiry. If you need something that lasts longer than
+a few hours, prefer a token resource with a concrete `metadata.expires` timestamp instead of cranking
+`--ttl` to very large values. Example (set an appropriate, bounded expiry):
+
+```yaml
+# token-long-lived.yaml
+kind: token
+version: v2
+metadata:
+  name: node-join-24h
+  expires: 2025-12-31T23:59:00Z
+spec:
+  join_method: token
+  roles: [Node]
+```
+
+```code
+$ tctl create -f token-long-lived.yaml
+```
+
+### Delegated joins
+
+Delegated joins avoid shipping any secret token in user-data. You create a named token resource
```

Comment:
We haven't mentioned shipping any secret token in user-data before this point, so it's not completely clear what this sentence is referring to.